packages/cubejs-materialize-driver/test/MaterializeDriver.test.ts

```diff
@@ -0,0 +1,120 @@
+import { MaterializeDBRunner } from '@cubejs-backend/testing';
+
+import { StartedTestContainer } from 'testcontainers';
+
+import { MaterializeDriver } from '../src';
+
+const streamToArray = require('stream-to-array');
```

Comment:
We are deprecating `stream-to-array`. Please use `streamToArray` from `@cubejs-backend/backend-shared`